docs/Inside_Arbitrum_Nitro/Inside_Arbitrum_Nitro.md

```diff
@@ -0,0 +1,527 @@
+---
+id: Inside_Arbitrum_Nitro
+title: Inside Arbitrum Nitro
+sidebar_label: Inside Arbitrum Nitro
```

Comment:
Don't seen this for the new docs (id is file name, label is set in docs repo). Can replace with header:

```
# Inside Arbitrum Nitro
```

Reply:
Good to standardize to filenames/links with all lowercase to avoid URL issues